scripts/rest-test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Runs a CLI-based integration test
+
+set -euxo pipefail
+export RUST_LOG=info
+
+source ./scripts/lib.sh
+source ./scripts/build.sh
+source ./scripts/setup-tests.sh
+./scripts/start-fed.sh
+export PEG_IN_AMOUNT=99999
+./scripts/pegin.sh $PEG_IN_AMOUNT 1 # peg in gateway
```

Comment:
We don't need line 12 because the gateway doesn't need any funds in this test. I don't think we need `export PEG_IN_AMOUNT=99999` either ...